data/xml/2018.iwslt.xml

```diff
@@ -0,0 +1,351 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2018.iwslt">
+  <volume id="1" ingest-date="2022-02-18">
+    <meta>
+      <booktitle>Proceedings of the 15th International Conference on Spoken Language Translation</booktitle>
+      <publisher>Association for Computational Linguistics</publisher>
```

Comment:
I don't think ACL was the publisher until last year or maybe 2020. This could also affect past volumes. Can you check with them?

Reply:
It might just be “IWSLT” itself…